aries-site/src/pages/components/tag.mdx

```diff
@@ -89,6 +93,10 @@ The experience for creating a Name-Value pair tag occurs in 2 steps: Create the
 
 Currently, assigning tags occurs within an editor experience in a [modal](/components/layer#side-drawer-modal) or [wizard](/templates/wizard). This helps to focus the task for the user. Tags are first selected and/or created, then assigned to the resource.
 
+<Example template>
```

Comment:
Not tied to this PR, but from a content standpoint, I'd think we'd lead with a simpler "create and assign tags" example first, followed by the more advanced "create and assign name-value tags".

Add this work to the tracking issue?

Reply:
Sounds good, I'll add it to the tracking issue.

Reply:
#2165